openmetadata-airflow-apis/src/openmetadata/api/rest_api.py

```diff
@@ -201,13 +200,13 @@ def test_connection() -> Response:
         except ValidationError as err:
             return ApiResponse.error(
                 status=ApiResponse.STATUS_BAD_REQUEST,
-                error=f"Request Validation Error parsing payload {json_request}. (Workflow)Source expected - {err}",
```

Comment:
should we double-check this in the `deploy` endpoint as well? If versions are aligned I am not sure this issue could happen though

Reply:
for example I am doing some changes now in airflow REST and seeing this 
![image](https://user-images.githubusercontent.com/35870520/175085618-5bbb0a17-ca0a-4113-8fa8-e37a59984ba1.png)

IDK if it might better be safe than sorry here

Reply:
ah actually we are just passing the service entity reference here, you can ignore me!!